hw/ip/otp_ctrl/data/otp_ctrl.hjson.tpl

```diff
@@ -336,6 +336,69 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "SECRET.DATA.SCRAMBLE",
+      desc: "Secret partitions are scrambled with a full-round PRESENT cipher."
+    }
+    { name: "PART.DATA.DIGEST",
+      desc: "Integrity of buffered partitions is ensured via a 64bit digest."
+    }
+    { name: "DAI.FSM.SPARSE",
+      desc: "The direct access interface FSM is sparsely encoded."
+    }
+    { name: "LCI.FSM.SPARSE",
+      desc: "The life cycle interface FSM is sparsely encoded."
+    }
+    { name: "KDI.FSM.SPARSE",
+      desc: "The key derivation interface FSM is sparsely encoded."
+    }
+    { name: "SCRMBL.FSM.SPARSE",
+      desc: "The scramble datapath FSM is sparsely encoded."
+    }
+    { name: "TIMER.FSM.SPARSE",
+      desc: "The background check timer FSMs are sparsely encoded."
+    }
+    { name: "DAI.CNT.REDUN",
+      desc: "The direct access interface address counter is duplicated."
+    }
+    { name: "KDI_SEED.CNT.REDUN",
+      desc: "The key derivation interface counter is duplicated."
+    }
+    { name: "KDI_ENTROPY.CNT.REDUN",
+      desc: "The key derivation entropy counter is duplicated."
+    }
+    { name: "LCI.CNT.REDUN",
+      desc: "The life cycle interface address counter is duplicated."
+    }
+    { name: "PART.CNT.REDUN",
+      desc: "The address counter of buffered partitions is duplicated."
+    }
+    { name: "SCRMBL.CNT.REDUN",
+      desc: "The srambling datapath counter is duplicated."
+    }
+    { name: "TIMER_INTEG.CNT.REDUN",
+      desc: "The background integrity check timer is duplicated."
+    }
+    { name: "TIMER_CNSTY.CNT.REDUN",
+      desc: "The background consistency check timer is duplicated."
+    }
+    { name: "TIMER.LFSR.REDUN",
+      desc: "The background check LFSR is duplicated."
+    }
+    { name: "FSM.ESCALATION",
+      desc: "All redundantly encoded FSMs are moved into an invalid state upon escalation."
+    }
+    { name: "FSM.LOCAL_ESC",
+      desc: "All redundantly encoded FSMs are moved into an invalid state upon local escalation."
+    }
+    { name: "PART.DATA.INTEGRITY",
+      desc: "All partition buffer registers are protected with ECC on 64bit blocks."
+    }
+    { name: "MACRO.DATA.INTEGRITY",
+      desc: "All OTP macro words are protected with ECC on 16bit words."
+    }
+    { name: "PART.DATA.BKGN_CHK",
```

Comment:
Also here `DATA` should probably be `DATA_REG` or `MEM` depending on the used storage type.

Reply:
Jep, it should probably be `DATA_REG`, since it is the buffered content that is being checked against what is present in memory.